app/models/event.py

```diff
@@ -62,6 +62,8 @@ class Privacy:
     is_promoted = db.Column(db.Boolean, default=False, nullable=False)
     is_demoted = db.Column(db.Boolean, default=False, nullable=False)
     is_chat_enabled = db.Column(db.Boolean, default=False, nullable=False)
+    is_document_enabled = db.Column(db.Boolean, default=False, nullable=False)
+    document_links = db.Column(db.JSON)
```

Comment:
Import `JSONB` data type `from sqlalchemy.dialects.postgresql import JSONB`. 
Document links column would look like this:
```
document_links = db.Column(JSONB)
```
The schema would probably remain the same.